<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WavegenDataDialog</class>
 <widget class="QDialog" name="WavegenDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wavegen settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Number of initial waveforms</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="nInitialWaves">
     <property name="maximum">
      <number>10000000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Number of runs to find deltabar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="nDeltabarRuns">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Maximum epochs</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="maxIterations">
     <property name="maximum">
      <number>10000000</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>Number of EE trajectories</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="nTrajectories">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_22">
     <property name="text">
      <string>Length of EE trajectories (2,4,8,16,32)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="trajectoryLength">
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>32</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QCheckBox" name="useBaseParameters">
     <property name="text">
      <string>Force use of preset parameter values</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Increase precision in epochs</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="precisionIncreaseEpochs">
     <property name="placeholderText">
      <string>Comma-separated integers</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Initial number of bins for bubble time</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QSpinBox" name="nBinsBubbleTime">
     <property name="maximum">
      <number>10000000</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Initial number of bins for bubble duration</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QSpinBox" name="nBinsBubbleDuration">
     <property name="maximum">
      <number>10000000</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Initial number of bins for voltage deviation</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QSpinBox" name="nBinsVoltageDeviation">
     <property name="maximum">
      <number>10000000</number>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Initial number of bins for voltage integral</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QSpinBox" name="nBinsVoltageIntegral">
     <property name="maximum">
      <number>10000000</number>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Initial number of bins for mean current</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QSpinBox" name="nBinsMeanCurrent">
     <property name="maximum">
      <number>10000000</number>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>Mean current maximum value</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QDoubleSpinBox" name="meanCurrent_max">
     <property name="suffix">
      <string> nA</string>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="3">
    <widget class="QCheckBox" name="mape_nClusters">
     <property name="text">
      <string>Use number of clusters as MAPE dimension</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="3">
    <widget class="QCheckBox" name="mape_clusterIndex">
     <property name="text">
      <string>Use cluster index as MAPE dimension</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WavegenDataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WavegenDataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
